pkg/scheduler/plugins/drf/drf.go

```diff
@@ -385,11 +385,6 @@ func (drf *drfPlugin) OnSessionOpen(ssn *framework.Session) {
 	})
 }
 
-func (drf *drfPlugin) updateNamespaceShare(namespaceName string, attr *drfAttr) {
```

Comment:
This should be because the namespace-based fair scheduling was not cleaned up before, and there is another place that can be updated together. The function name in the comment content is wrong, as follows:

https://github.com/volcano-sh/volcano/blob/d4458ffe81207749f04727873d37544d7138152d/pkg/scheduler/plugins/drf/drf.go#L438-L439

Reply:
I find no comment above this func in master.